commands/js.js

```diff
@@ -0,0 +1,229 @@
+            required: true,
+        })
+        .option('initFunction', {
+            desc: 'Initialization method',
+            type: 'string',
+        })
+        .option('jsvm', {
+            desc: 'JSVM enclave contract id',
+            type: 'string',
+            default: null,
+        }),
+    handler: exitOnError(deploy),
+};
+
+const js_remove = {
+    command: 'remove [accountId]',
+    desc: 'Remove a JSVM contract from the network',
+    builder: (yargs) => yargs
+        .option('accountId', {
+            desc: 'The id of the account that will be removed from the enclave',
+            type: 'string',
+            required: true,
+        })
+        .option('gas', {
+            desc: 'Gas used to remove the contract from the enclave',
+            type: 'number',
+            default: DEFAULT_FUNCTION_CALL_GAS,
+        })
+        .option('jsvm', {
+            desc: 'JSVM enclave contract id',
+            type: 'string',
+            default: null,
+        }),
+    handler: exitOnError(remove),
+};
+
+const js_call = {
+    command: 'call [contractId] [methodName] [args] [gas] [deposit]',
+    desc: 'Call a method on a contract',
+    builder: (yargs) => yargs
+        .option('accountId', {
+            desc: 'Unique identifier for the account that will be used to sign this call',
+            type: 'string',
+            required: true,
+        })
+        .option('args', {
+            desc: 'Arguments to the contract call, in JSON format by default (e.g. \'{"param_a": "value"}\')',
+            type: 'string',
+            default: '',
+        })
+        .option('gas', {
+            desc: 'Gas used to make a call into the contract from the enclave',
+            type: 'number',
+            default: DEFAULT_FUNCTION_CALL_GAS,
+        })
+        .option('deposit', {
+            desc: 'Deposit to maintain the contract storage on the enclave',
+            type: 'string',
+            default: '0',
+        })
+        .option('depositYocto', {
+            desc: 'Deposit (in Yocto Near) to maintain the contract storage on the enclave',
+            type: 'string',
+            default: null,
+        })
+        .option('jsvm', {
+            desc: 'JSVM enclave contract id',
+            type: 'string',
+            default: null,
+        }),
+    handler: exitOnError(call),
+};
+
+module.exports = {
+    command: 'js <command> <command-options>',
+    desc: 'Add an access key to given account',
+    builder: (yargs) => yargs
+        .command(js_deploy)
+        .command(js_remove)
+        .command(js_call),
+};
+
+// Encode JSVM related arguments into base64 for use in the enclave
+function base64_encode_args(contractId, functionName, args) {
+    let buf = Buffer.concat([
+        Buffer.from(contractId),
+        Buffer.from([0]),
+        Buffer.from(functionName),
+        Buffer.from([0]),
+        Buffer.from(args)]
+    );
+    return Buffer.from(buf.toString('base64'), 'base64');
```

Comment:
This is a invariant operation. 
```
> Buffer.from([0])
<Buffer 00>
> Buffer.from([0]).toString('base64')
'AA=='
> Buffer.from(Buffer.from([0]).toString('base64'), 'base64')
<Buffer 00>

```


it's equivalent to simply return Buffer:
```suggestion
    return buf
```